Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,15 @@
 import React from 'react';
+import CardBanner from './CardBanner';
+import CardContent from './CardContent';
 import './Card.css';
```

Comment:
when you're only using one file for css, you can import it only into the top-level component of the chain that you're styling with it.